### ISSUE TITLE
Unable to save Quick-Pad files when it started or created via Files App

### ISSUE THREAD

**Issue by BitmapBrother** [yaichenbaum/Quick-Pad#456]
Hello again Yair A. here on GitHub...

Today we had a short mail contact together and you asked me in this conversation to open an issue here on GitHub for the saving bug of the Quick-Pad app. Here I am and here is the bug discription again:

The problem with Quick Pad is that I’m not able to save anything when I open a Quick Pad file in the Files App. It doesn’t matter if the Quick Pad file is a new one or an already existing file that I want to edit.  I can’t save any changes.

When I open the same Quick Pad files in Windows Explorer saving changes isn’t a problem at all. Quick Pad works here as expected.

Best regards

**Comment by BitmapBrother** [comment]
Hello and thank you for the Information…

The discribed bug is really gone and I can save now edited QuickPad files that I try to manage within the Files app. Well done!

Unfortunately I can not test if I can create new QuickPad files within the Files app because the Files app is very buggy again and this time I am not able to create a QuickPad file within the Files app. The Right-click-menu function leads always to a hard crash of the Files app. By the way the current version of the Files app shows no icons of any file types and also no icons of applications but only folder icons and otherwise blank paper icons for any other files.   

Still one comment about the functionality of the QuickPad app:
Now the language settings are wrong in the QuickPad app. I’ve set the language to German but QuickPad shows most of the informations in Spanish and some in English to me.    

Best regards
Michel

Von: Yair Aichenbaum
Gesendet: Dienstag, 11. Mai 2021 16:33
An: yaichenbaum/Quick-Pad
Cc: BitmapBrother; Author
Betreff: Re: [yaichenbaum/Quick-Pad] Unable to save Quick-Pad files when itstarted or created via Files App (#456)

This issue should be resolved in main

**Comment by yaichenbaum** [comment]
@BitmapBrother English translations will be used for strings that weren't localized. Would you like to update the German translation?

**Comment by BitmapBrother** [comment]
Yes, I would like to update the German translation. But how can I do it? QuickPad shows no reactions when I change the language in the settings. It does not matter which language I choose. QuickPad displays the whole informations all the time in Portuguese (not in Spanish…sorry)… 


Von: Yair Aichenbaum
Gesendet: Mittwoch, 12. Mai 2021 21:36
An: yaichenbaum/Quick-Pad
Cc: BitmapBrother; Mention
Betreff: Re: [yaichenbaum/Quick-Pad] Unable to save Quick-Pad files when itstarted or created via Files App (#456)

@BitmapBrother English translations will be used for strings that weren't localized. Would you like to update the German translation?

**Comment by yaichenbaum** [comment]
You need to restart Quick Pad for the change to take affect.

**Comment by BitmapBrother** [comment]
I’ve restarted the application a few times but it takes no effect when I choose German. I’ve tested some other language settings:

• Hebrew -> it works
• Spanish -> it works
• Arabic -> it works
• Portuguese -> it works
• French -> it works
• Tamil -> it works
• And so on…
• But German -> no Chance

Von: Yair Aichenbaum
Gesendet: Mittwoch, 12. Mai 2021 22:36
An: yaichenbaum/Quick-Pad
Cc: BitmapBrother; Mention
Betreff: Re: [yaichenbaum/Quick-Pad] Unable to save Quick-Pad files when itstarted or created via Files App (#456)

You need to restart Quick Pad for the change to take affect.

**Comment by yaichenbaum** [comment]
Can you try reinstalling Quick Pad?

**Comment by BitmapBrother** [comment]
Hi Yair,

I‘ve reinstalled the QuickPad app but the behaviour of the app does not change. Please check the attachment… 

Regards
Michel

Von: Yair Aichenbaum
Gesendet: Donnerstag, 13. Mai 2021 02:41
An: yaichenbaum/Quick-Pad
Cc: BitmapBrother; Mention
Betreff: Re: [yaichenbaum/Quick-Pad] Unable to save Quick-Pad files when itstarted or created via Files App (#456)

Can you try reinstalling Quick Pad?

**Comment by yaichenbaum** [comment]
@BitmapBrother The issue with the translations should be resolved in `main`.
